package.json

```diff
@@ -22,7 +22,7 @@
     "@angular/platform-browser": "^7.1.3",
     "@angular/platform-browser-dynamic": "^7.1.3",
     "@angular/router": "^7.1.3",
-    "@tinymce/tinymce-angular": "^2.4.1",
+    "@tinymce/tinymce-angular": "^2.5.0",
```

Comment:
Unless there is code which depends on a new and improved version of a dependency, we should keep version changes in their own PRs. Just for the sake of cleanliness.